src/utilities/prediction/macroEyes.js

```diff
@@ -6,15 +6,12 @@
  * - Update suggested values in requisition item list
  *
  */
+import { ToastAndroid } from 'react-native';
 import { useEffect, useState } from 'react';
 
 import { UIDatabase } from '../../database';
 import { SETTINGS_KEYS } from '../../settings';
-
-const API_URL =
-  UIDatabase.getSetting(SETTINGS_KEYS.ME_PREDICTION_API_URL) || 'http://civapi.dev.macro-eyes.com';
-
-const API_KEY = UIDatabase.getSetting(SETTINGS_KEYS.ME_PREDICTION_API_KEY);
```

Comment:
I was looking at some react code somewhere else and thinking about this again. It's not immediately obvious because these constants are simply used inside the methods below, but they've probably worked functionally like the database singleton pattern below:

https://github.com/openmsupply/mobile/blob/1f849502e6fb6cbb7984ca17b83522643e30ac32/src/database/BaseDatabase.js#L1-L21